src/__tests__/loadable-components/intersection-observer.test.js

```diff
@@ -7,6 +7,7 @@ const { IntersectionObserver, makeElementsVisible } = require('../../__mocks__/I
 global.IntersectionObserver = IntersectionObserver
 
 const loadableVisiblity = require('../../loadable-components')
+const trackedElements = require('../../tracked_elements').default
```

Comment:
It looks like this is primarily being used for the tests. Would you be able to modify the `IntersectionObserver` mock to include this behaviour instead?